rclpy_components/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rclpy_components</name>
+  <version>0.0.0</version>
```

Comment:
For maintenance reasons, this should match the version of other packages in this repository.
```suggestion
  <version>1.1.0</version>
```